src/doc/rustc/src/command-line-arguments.md

```diff
@@ -42,11 +42,11 @@ This flag prints out various information about the compiler.
 
 ## `-g`: include debug information
 
-A synonym for `-C debug-level=2`.
+A synonym for `-C debuglevel=2`, for more see [here](codegen-options/index.html#debuginfo).
```

Comment:
Looking at the output of `rustc +stable -C help`, it looks like this should actually be `debuginfo`.